cmd/cmd_controller.js

```diff
@@ -478,7 +476,6 @@ class EmbarkController {
         engine.startService("deployment");
         engine.startService("storage");
         engine.startService("codeGenerator");
-        engine.startService("namingSystem");
```

Comment:
Since we put namingSystem is deploymentService, can you remove the namingSystem function in engine, please.